### ISSUE TITLE
Running pip install robotframework-ride  fails on windows 11

### ISSUE THREAD

**Issue by ryanrosello-og** [robotframework/RIDE#2517]
**My environment:**

Windows 11
Windows Terminal > Powershell
 ~  pip --version
pip 22.3.1 from C:\Users\ryanr\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip (python 3.11)

 ~  python --version
Python 3.11.1

When trying to run `pip install robotframework-ride`, I encounter the error below:

```
Collecting robotframework-ride
  Downloading robotframework-ride-1.7.4.2.tar.gz (846 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 846.5/846.5 kB 5.4 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
Collecting wxPython<=4.0.7.post2
  Downloading wxPython-4.0.7.post2.tar.gz (68.9 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 68.9/68.9 MB 1.4 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
Collecting PyPubSub
  Downloading Pypubsub-4.0.3-py3-none-any.whl (61 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 61.4/61.4 kB 3.2 MB/s eta 0:00:00
Collecting Pygments
  Downloading Pygments-2.14.0-py3-none-any.whl (1.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.1/1.1 MB 4.2 MB/s eta 0:00:00
Collecting Pywin32
  Downloading pywin32-305-cp311-cp311-win_amd64.whl (12.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 12.1/12.1 MB 5.7 MB/s eta 0:00:00
Collecting pillow
  Downloading Pillow-9.4.0-cp311-cp311-win_amd64.whl (2.5 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.5/2.5 MB 5.6 MB/s eta 0:00:00
Collecting six
  Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)
Collecting numpy
  Downloading numpy-1.24.1-cp311-cp311-win_amd64.whl (14.8 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 14.8/14.8 MB 4.5 MB/s eta 0:00:00
Installing collected packages: Pywin32, six, PyPubSub, Pygments, pillow, numpy, wxPython, robotframework-ride
  DEPRECATION: wxPython is being installed using the legacy 'setup.py install' method, because it does not have a 'pyproject.toml' and the 'wheel' package is not installed. pip 23.1 will enforce this behaviour change. A possible replacement is to enable the '--use-pep517' option. Discussion can be found at https://github.com/pypa/pip/issues/8559
  Running setup.py install for wxPython ... error
  error: subprocess-exited-with-error

  × Running setup.py install for wxPython did not run successfully.
  │ exit code: 1
  ╰─> [43 lines of output]
      C:\Users\ryanr\AppData\Local\Programs\Python\Python311\Lib\site-packages\setuptools\dist.py:771: UserWarning: Usage of dash-separated 'license-file' will not be supported in future versions. Please use the underscore name 'license_file' instead
        warnings.warn(
      C:\Users\ryanr\AppData\Local\Programs\Python\Python311\Lib\site-packages\setuptools\config\setupcfg.py:508: SetuptoolsDeprecationWarning: The license_file parameter is deprecated, use license_files instead.
        warnings.warn(msg, warning_class)
      C:\Users\ryanr\AppData\Local\Programs\Python\Python311\Lib\site-packages\setuptools\dist.py:317: DistDeprecationWarning: use_2to3 is ignored.
        warnings.warn(f"{attr} is ignored.", DistDeprecationWarning)
      running install
      C:\Users\ryanr\AppData\Local\Programs\Python\Python311\Lib\site-packages\setuptools\command\install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
        warnings.warn(
      running build
      Will build using: "C:\Users\ryanr\AppData\Local\Programs\Python\Python311\python.exe"
      3.11.1 (tags/v3.11.1:a7a450f, Dec  6 2022, 19:58:39) [MSC v.1934 64 bit (AMD64)]
      Python's architecture is 64bit
      cfg.VERSION: 4.0.7.post2

      Running command: build
      Running command: build_wx
      Command '"C:\Users\ryanr\AppData\Local\Programs\Python\Python311\python.exe" -c "import distutils.msvc9compiler as msvc; mc = msvc.MSVCCompiler(); mc.initialize(); print(mc.cc)"' failed with exit code 1.
      Traceback (most recent call last):

        File "<string>", line 1, in <module>

        File "C:\Users\ryanr\AppData\Local\Programs\Python\Python311\Lib\site-packages\setuptools\_distutils\msvc9compiler.py", line 403, in initialize

          vc_env = query_vcvarsall(VERSION, plat_spec)

                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

        File "C:\Users\ryanr\AppData\Local\Programs\Python\Python311\Lib\site-packages\setuptools\_distutils\msvc9compiler.py", line 281, in query_vcvarsall

          raise DistutilsPlatformError("Unable to find vcvarsall.bat")

      distutils.errors.DistutilsPlatformError: Unable to find vcvarsall.bat
      Finished command: build_wx (0.413s)
      Finished command: build (0.413s)
      WARNING: Building this way assumes that all generated files have been
      generated already.  If that is not the case then use build.py directly
      to generate the source and perform the build stage.  You can use
      --skip-build with the bdist_* or install commands to avoid this
      message and the wxWidgets and Phoenix build steps in the future.

      "C:\Users\ryanr\AppData\Local\Programs\Python\Python311\python.exe" -u build.py build
      Command '"C:\Users\ryanr\AppData\Local\Programs\Python\Python311\python.exe" -u build.py build' failed with exit code 1.
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: legacy-install-failure

× Encountered error while trying to install package.
╰─> wxPython

note: This is an issue with the package mentioned above, not pip.
hint: See above for output from the failure.
```

**Comment by axiom41** [comment]
I guess it depends on Incompatibilities between versions: Python 3.11.x is still not supported at all and the stable but old RIDE 1.7.4.2 is compatible until Python 3.7.x
I would suggest you to downgrade Python to 3.10.x and use the newest RIDE 2.0b3 (not yet a final, but fairly stable) by following instructions written here on the "Code" tab

**Comment by HelioGuilherme66** [comment]
The version you are trying to install (1.7.4.2) is not compatible with Python >=3.8.
The installation is trying to install wxPython.
I recommend for you to install wxPython 4.2.0 first.

You can attempt to install RIDE from current master (because has a fix on 2.0b3).
